config/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class CareManager(models.Manager):
```

Comment:
What happens when we want to query of `is_active` False? 
If I write a query => `Patients.objects.filter(is_active=False)`, would that override the earlier `is_active=True`?